libs/tasks/feature/src/lib/tasks-feature/tasks-feature.component.ts

```diff
@@ -1,30 +1,78 @@
-import { Component, OnInit } from '@angular/core';
-import { Task, TasksFacade } from '@todo-workspace/tasks/data-access';
-import { Observable } from 'rxjs';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import {
+  SnackbarService,
+  TasksFacade,
+} from '@todo-workspace/tasks/data-access';
+import { Subject } from 'rxjs';
 import { MatDialog } from '@angular/material/dialog';
 import { TaskFormDialogComponent } from '@todo-workspace/tasks/ui-task-form-dialog';
+import { Task } from '@todo-workspace/tasks/domain';
+import { takeUntil } from 'rxjs/operators';
 
 @Component({
   selector: 'todo-workspace-tasks-feature',
   templateUrl: './tasks-feature.component.html',
   styleUrls: ['./tasks-feature.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class TasksFeatureComponent implements OnInit {
-  tasks$: Observable<Task[]>;
-  constructor(private tasksFacade: TasksFacade, private matDialog: MatDialog) {}
+export class TasksFeatureComponent implements OnInit, OnDestroy {
+  tasks: Task[] = [];
+  readonly destroy$ = new Subject<void>();
+  readonly todoPage$ = this.tasksFacade.todoPage$;
+  readonly donePage$ = this.tasksFacade.donePage$;
+  readonly loading$ = this.tasksFacade.loading$;
+
+  constructor(
+    public tasksFacade: TasksFacade,
+    private matDialog: MatDialog,
+    private snackbar: SnackbarService
+  ) {}
+
+  ngOnDestroy(): void {
+    this.destroy$.next();
+    this.destroy$.unsubscribe();
+  }
 
   ngOnInit(): void {
     this.tasksFacade.init();
-    this.tasks$ = this.tasksFacade.tasks$;
+
+    this.tasksFacade.error$
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((value) => {
+        this.snackbar.open(value);
+      });
+
+    this.tasksFacade.tasks$
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((value) => (this.tasks = value));
```

Comment:
generally, you can change the `value` name to the `tasks` name. It is more readable